runtime/ops/worker_host.rs

```diff
@@ -442,6 +445,10 @@ fn op_create_worker(
   if use_deno_namespace {
     super::check_unstable(state, "Worker.deno.namespace");
   }
+  let worker_type = args.worker_type;
+  if let WebWorkerType::Classic = worker_type {
+    super::check_testing_features(state, "Worker.classic");
```

Comment:
I would rather that this just errors out gracefully with `"classic" workers are not supported` not supported dom exception.

Reply:
I'll simplify `TestingFeatureChecker` as well: since it's only ever read once, the code to borrow it can simply be put inline rather than on a function like `check_unstable`.